bin/localicious-render

```diff
@@ -1,59 +1,86 @@
 #!/usr/bin/env node
 const path = require("path");
 const program = require("commander");
+const keywords = require("../src/model/keywords");
 const validateAndRender = require("../src/actions/validateAndRender");
 
 const commaSeparatedList = value => {
   return value.split(",");
 };
 
-program
-  .option("-l --languages <languages>", "output languages", commaSeparatedList)
-  .option("-p --platforms <platforms>", "target platforms", commaSeparatedList)
-  .parse(process.argv);
+const hasValidLanguages = languages => {
```

Comment:
Since we're not really validating the languages but rather just check whether they are defined, would suggest:

```suggestion
const hasLanguages = languages => {
```